src/bundle/Resources/public/scss/_switcher.scss

```diff
@@ -1,15 +1,17 @@
 .ibexa-tab-switcher {
-    padding: calculateRem(4px);
-    border-radius: $ibexa-border-radius;
-    background-color: $ibexa-color-light-300;
-    display: flex;
+    &__list {
+        padding: calculateRem(4px);
+        border-radius: $ibexa-border-radius;
+        background-color: $ibexa-color-light-300;
+        display: flex;
+    }
 
-    &__item {
+    &__tab {
         height: calculateRem(40px);
         padding: calculateRem(10px);
         border-radius: $ibexa-border-radius;
         border: 0;
-        flex-basis: 100%;
+        flex-basis: auto;
```

Comment:
If it is not changing any flex-basiss value it can be removed since auto is default value